kafka-rest/src/test/java/io/confluent/kafkarest/integration/MetadataAPITest.java

```diff
@@ -189,4 +177,25 @@ public void testPartitionsList() throws InterruptedException {
         Errors.PARTITION_NOT_FOUND_MESSAGE,
         Versions.KAFKA_V2_JSON);
   }
+
+  private void verifyTopicGet() {
+    Response response = request("/topics").get();
+    assertOKResponse(response, Versions.KAFKA_V2_JSON);
+    final List<String> topicsResponse =
+        tryReadEntityOrLog(response, new GenericType<List<String>>() {});
+    assertEquals(Arrays.asList(topic1Name, topic2Name), topicsResponse);
+  }
+
+  private void verifyPartitionGet(String topicName, int size, int numPartitions) {
```

Comment:
These parameters seem confusing to me. In the current code, it looks like both of them refer to the number of topic partitions. Was the idea to have `size` be `numPartitions`, and to have `numPartitions` be `numReplicas`, similar to the way the `createTopic` method works?